Podfile

```diff
@@ -6,8 +6,8 @@ use_frameworks!
 
 def commonpods
   pod 'IP-UIKit-Wisdom'
-    pod 'RxSwift', '~> 3.0'
-    pod 'RxCocoa', '~> 3.0'
+    pod 'RxSwift', :git => 'https://github.com/ReactiveX/RxSwift.git', :branch => 'swift4.0'
+    pod 'RxCocoa', :git => 'https://github.com/ReactiveX/RxSwift.git', :branch => 'swift4.0'
```

Comment:
Is this the right branch to point to? [RxSwift's README](https://github.com/ReactiveX/RxSwift) is pointing to a different branch for Swift 4. Also [CocoaPods' RxSwift page](https://cocoapods.org/?q=rxswift) says the latest version is 4.0.0-beta.0 which is Swift 4 compatible (I just tested); should we point directly to that instead?

Reply:
Looks valid. I think they changed their branch between then and now.

Reply:
Damn, they haven't pushed `RxTest` 4 version.

Reply:
Wait no it works